react-native-reanimated.d.ts

```diff
@@ -636,6 +636,11 @@ declare module 'react-native-reanimated' {
     }): void;
     export function addWhitelistedUIProps(props: { [key: string]: true }): void;
 
+    export class Keyframe {
+      constructor(definitions: Map<number, KeyFrameProps[]>);
```

Comment:
`KeyFrameProps` is defined in `commonTypes.ts` and I believe we don't include it in the d.ts so this will probably produce an error.